include/mxnet/c_api.h

```diff
@@ -51,6 +51,10 @@ typedef void *RecordIOHandle;
 /*! \brief handle to MXRtc*/
 typedef void *RtcHandle;
 
+namespace mxnet {
+class NDArray;
```

Comment:
This is a C API, so please avoid include C++ object here, otherwise it can break for some FFI